src/components/Dropdown/Dropdown.tsx

```diff
@@ -168,7 +168,9 @@ export class Dropdown extends React.PureComponent<IDropdownProps, IDropdownState
                             gapSpace={0}
                             doNotLayer={false}
                             targetElement={this._dropDown}
-                            directionalHint={DirectionalHint.bottomLeftEdge}
+                            directionalHint={this.props.calloutDirectionalHint ?
```

Comment:
It would be better to use default props instead of this check in the render method.